data/metainfo.appdata.xml.in

```diff
@@ -7,7 +7,10 @@
 
   <name>Clairvoyant</name>
   <summary>Ask questions, get psychic answers</summary>
-  <developer_name>Cassidy James Blaede</developer_name>
+  <developer_name translatable="no">Cassidy James Blaede</developer_name>
+  <developer id="github.com">
```

Comment:
```suggestion
  <developer id="cassidyjames.com">
```

I'm not GitHub; this should be a unique ID under my control (a "web URL" as recommended by the [AppStream docs](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-developer)).